connection.go

```diff
@@ -127,6 +130,11 @@ type ConnectionOptions struct {
 	// A static set of prefixes to check against the remote process name when
 	// setting up a connection. Check matches with RemoteProcessPrefixMatches().
 	CheckedProcessPrefixes []string
+
+	// QoS class name to be marked on outbound connection. Supply a DiffServ
+	// Compliant name, AF11 .. etc,  or "LOWDELAY", "THROUGHPUT", "RELIABILTIY"
```

Comment:
rather than quoting the `"LOWDELAY"` and `"THROUGHPUT"`, I think we should just mention `tos.LowDelay` and `tos.Throughput`